core/src/main/java/nl/tudelft/contextproject/core/entities/Colour.java

```diff
@@ -1,21 +1,103 @@
 package nl.tudelft.contextproject.core.entities;
 
 import com.badlogic.gdx.graphics.Color;
-import lombok.Getter;
 
-/**
- * Created by Mitchell on 12-5-2015.
- */
+import java.util.Collection;
+import java.util.Iterator;
+
 public enum Colour {
 
-    RED(255, 0, 0), YELLOW(255, 255, 0), BLUE(0, 0, 255), LIGHTGREEN(0, 255, 0), GREEN(34, 170, 34), GREY(127, 127, 127),
-    DAKBLUE(0, 0, 255), PURPLE(128, 0, 128), PINK(250, 105, 180), BROWN(139, 69, 69), ORANGE(255, 165, 0),
-    SALMON(255, 181, 153), BLACK(0, 0, 0);
+    RED(1, 0, 0, -16776961),
+    YELLOW(1, 1, 0, -65281),
+    BLUE(0, 0, 1, 65535),
+    GREEN(34 / 255f, 170 / 255f, 34 / 255f, 581575423),
+    PURPLE(128 / 255f, 0, 128 / 255f, -2147450625),
+    ORANGE(255 / 255f, 165 / 255f, 0, -5963521),
+    BLACK(0, 0, 0, 255);
+
+    private Color libgdxColor;
```

Comment:
these field could be declared final, mutability in enum classes is generally best if avoided.